### PR TITLE
Convert dtype to int instead of np.uint8 for visualising large task graphs

### PR DESCRIPTION
**Pull request by phofl**
- [ ] Closes #xxxx
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

cmap returns np.uint8 since numpy 2, but this raises in _colorize for larger tasks graphs when visualising order stuff. I ran into this a few times with larger graphs before trimming them down.

This is the problem.

```
i = sum(v * 256 ** (len(t) - i - 1) for i, v in enumerate(t))
```

Don't really know how and if we should test this though :(

cc @fjetter

### CODE REVIEW

File: dask/base.py [hunk 1]
```diff
@@ -853,7 +853,10 @@ def label(x):
             values = o
         if maxval is None:
             maxval = max(1, max(values.values()))
-        colors = {k: _colorize(cmap(v / maxval, bytes=True)) for k, v in values.items()}
```
Comment (fjetter) [inline dask/base.py:856]: there's a similar _colorize call five lines below

**Comment by fjetter** [comment]
`_colorize((0, 0, np.uint8(2**8-1)))` works for me (both numpy <1 and >=2) but I'm happy to merge like this if it fixes the problem you've seen

**Comment by phofl** [comment]
Thanks for pointing out the other place!


I took a closer look what was raising, and an input like

```
(np.uint8(253), np.uint8(219), np.uint8(203), np.uint8(255))
```

will raise, this can happen with val 1, maxval 7

We actually have a graph in dask-order that forces this, so added a visualise call there

**Comment by fjetter** [comment]
> FAILED dask/tests/test_order.py::test_visualize_int_overflow - ModuleNotFoundError: No module named 'matplotlib'

**Pull request merged**